File: CloudDataProtection/CloudDataProtection.Services.BackupConfigurationService/Dto/CreateBackupConfigurationResult.cs
using System;
using CloudDataProtection.Services.Subscription.Entities;

namespace CloudDataProtection.Services.Subscription.Dto
{
    public class CreateBackupConfigurationResult
    {
        public int Id { get; set; }
        
        public DateTime CreatedAt { get; set; }
        
        public BackupFrequency Frequency { get; set; }
        
        public int Hour { get; set; }
        
        public int Minute { get; set; }

        public string Description => $"Backup my data {Frequency.ToString().ToLower()} at {Hour.ToString("00")}:{Minute.ToString("00")}";
    }
}

File: cloud-data-protection/src/services/result/backupConfiguration/createBackupConfigurationResult.ts
import {BackupFrequency} from "entities/backupFrequency";

interface CreateBackupConfigurationResult {
    id: number;
    createdAt: Date;
    frequency: BackupFrequency;
    hour: number;
    minute: number;
    description: string;
}

export default CreateBackupConfigurationResult;